modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -435,6 +554,30 @@ FROM <catalog-name>.ClickEvent_key_value;
 +------------------------------------------------------------------------------+
 ----
 
+== Dead-letter queue
+
+Errors may occur when writing records to the Iceberg table; for example, if the record does not match the topic schema, or you do not use the Schema Registry wire format and the schema is not found, or if an Avro or Protobuf data type cannot be translated to an Iceberg type.
+
+If Redpanda encounters an error while writing a record to the Iceberg table, the record is dropped by default. To enable a dead-letter queue (DLQ) for these records, configure the config_ref:redpanda.iceberg.invalid.record.action,true,properties/topic-properties[] topic property. Redpanda writes these records to a separate Iceberg table named `<topic-name>~dlq`.
```

Comment:
We default to DLQ starting 25.1!

Reply:
I don't understand the syntax for config_ref in this documentation but the property can be set to "drop" or "dlq_table". We also have an equivalent cluster wide config `iceberg_invalid_record_action` which is used if per topic override is not set.